### PR TITLE
Updated from OSX to macOS

### PR DESCRIPTION
**Pull request by SimoneBovi**
Updated all the occurrences of OSX with macOS.

### CODE REVIEW

**Comment by yahiheb** [comment]
LGTM

File: docs/FAQ/FAQ-Introduction.md [hunk 1]
```diff
@@ -276,7 +276,7 @@ For the complete explanation please read [ZeroLink: The Bitcoin Fungibility Fram
 
 - 64-bit architecture
 - Windows 7 or newer
```
Comment (lontivero) [inline docs/FAQ/FAQ-Introduction.md]: Can you take the opportunity and specify `Windows 7 SP1+`?

Comment (SimoneBovi) [reply docs/FAQ/FAQ-Introduction.md]: Done

**Comment by SimoneBovi** [comment]
For now I just committed an update from "Windows 7 or newer" to "Windows 7 SP1+" .

@lontivero I agree with you. Maybe we can merge this in the meantime and open a new PR for the section you mentioned?

**Pull request merged**